docs/pacific/New-Caledonia/Enroute.md

```diff
@@ -6,10 +6,44 @@
 
 ## Positions
 
-| Name                    | Login     | Callsign         | Frequency |
-| ----------------------- | --------- | ---------------- | --------- |
-| Noumea Control| 	NWWW_CTR| 	Tontouta Control	| 128.200| 
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Tontouta Approach North|NWWW| Tontouta Control  | 128.200         | NWWW_CTR          |
 
 ## Airspace
+### NFFF Release
+When NWWW is online, NFFF Class A airspace `F245`-`UNL` within the New Caledonia sector is released from NFFF to NWWW, unless coordinated otherwise.
 
-Reserved.
\ No newline at end of file
+## Coordination
+### NWWW / NWWW TCU
+#### Arrivals
+Voiceless coordination is in place from NWWW/NFFF to NWWW TCU given the arrival has been assigned the standard assignable level and depending on the duty runway:
+
+|Runway|Requirement|
+|------|-----------|
+|11|Assigned a STAR|
+|29|Tracking to LTO VOR|
+
+All other aircraft must be coordinated prior to 20nm from the boundary.
+
+The standard assignable level from NWWW to NWWW TCU is `A100`.
+
+#### Departures
+Voiceless coordination is in place from NWWW TCU to NWWW/NFFF for aircraft assigned the standard assignable level and tracking via a procedural SID terminus.
+
+The standard assignable level from NWWW TCU to NWWW is `F240`.
+
+### NWWW / NVVV
```

Comment:
Is surveillance coverage expected for this handoff? If yes, change parameter should be 50nm. If no, then 15 mins heads-up

Reply:
Discussed via PM, leaving as 25nm.